src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -53,7 +55,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                 {
                     // what do we do if there's no master?
                     TargetBranch = x.FirstOrDefault(b => b.Name == "master");
-                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head.FriendlyName);
+
+                    // 'repo.Head' will be null in unit tests.
+                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head?.FriendlyName);
```

Comment:
I dislike changing the code to cater to unit test assumptions, it would be better if the unit tests reflect the data that's supposed to be there. If `Head` is null, that means that the repository is not initialized/invalid (and this might be a thing we have to check before we actually get here) so we'd never get to this point at all. Can we change the unit tests to set the repo head properly?